File: sentimental-mario-less/mario.py
# TODO

from cs50 import get_int
# ask user for height input


def main():
    # for each row
    height = get_height()
    for row in range(height):
        # introduces space to the left
        for space in range(height - 1 - row):
            print(f" ", end="")
        # introduces hashes to the right
        for column in range(row + 1):
            print(f"#", end="")
        # move on to the next row
        print(f"\n", end="")

def get_height():
    while True:
        height = get_int("Height: ")
        if height > 0 and height < 9:
            return height


main()

File: mario-less/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height;
    do
    {
        height = get_int("Height: ");
    }
    while (height < 1 || height > 8);
// For each row
    for (int row = 0; row < height; row++)
    {
        // Introduce spaces to the left
        for (int space = 0; space < height - 1 - row; space++)
        {
            printf(" ");
        }
        // Introduce hashes to the right
        for (int column = 0; column <= row; column++)
        {
            printf("#");
        }
        // Move to next row
        printf("\n");
    }

}
